examples/js/renderers/CSS3DRenderer.js

```diff
@@ -10,6 +10,7 @@ THREE.CSS3DObject = function ( element ) {
 
 	this.element = element;
 	this.element.style.position = 'absolute';
+	this.element.style.pointerEvents = 'auto';
```

Comment:
CSS `pointer-events` is an inherits-property. this will affects all descendants such as camera element, CSS3Object elements.